docs/user-guide/faq.md

```diff
@@ -58,3 +58,20 @@ Before version 22.12, the CLI flag was `--experimental-solver`.
 We recommend you upgrade to `conda` 22.11 or above, and then `conda-libmamba-solver` 22.12 or above.
 
 See the [22.12.0 announcement post](https://github.com/conda/conda-libmamba-solver/releases/tag/22.12.0) for more details on how to upgrade.
+
+## I have an older `conda` and I can't install `conda-libmamba-solver`
+
+Since older `conda` versions only supported the `classic` solver, you might run into solver conflicts or too long installations if your `base` environment is too constrained. This becomes a "chicken-and-egg" problem where you'd need `conda-libmamba-solver` to update to a more recent `conda` with `conda-libmamba-solver`.
+
+Fortunately, there's a workaround thanks to the `conda-standalone` project. This is a single binary that bundles recent `conda` versions, with `conda-libmamba-solver` included. It's not a substitute for the full `conda` UX but it can help bootstrap and rescue conda installations.
+
+1. Download the most recent `conda-standalone` from its [Github Releases page](https://github.com/conda/conda-libmamba-solver/releases/latest). Make sure to pick the one for your operating system and platform. Once downloaded, rename it as `conda.exe`.
```

Comment:
```suggestion
1. Download the most recent `conda-standalone` from its [Github Releases page](https://github.com/conda/conda-standalone/releases/latest). Make sure to pick the one for your operating system and platform. Once downloaded, rename it as `conda.exe`.
```

The assets haven't been uploaded to the latest tag though?

Could we not use `conda.exe` on Posix? I think it'll confuse people

Reply:
If we make it `conda` the `conda` shell function will take precedence, I think. Also the internal filename in the conda-standalone bundle is `conda.exe`, so it's been around for a while... Let me see if we can express this succinctly.

Reply:
> The assets haven't been uploaded to the latest tag though?

I usually do this manually a few days after.